# pwnxy @ function > 
0x0000000000401040 in puts@plt ()
[#] DBG  : 36
[#] DBG  : 135
[#] DBG  : 36
[#] DBG  : 135
Python Exception <class 'AttributeError'> 'NoneType' object has no attribute 'fullname': 
Python Exception <class 'gdb.error'> Error occurred in Python: 'NoneType' object has no attribute 'fullname': 
# pwnxy @ function > 
0x0000000000401044 in puts@plt ()
[#] DBG  : 36
[#] DBG  : 135
[#] DBG  : 36
[#] DBG  : 135
Python Exception <class 'AttributeError'> 'NoneType' object has no attribute 'fullname': 
Python Exception <class 'gdb.error'> Error occurred in Python: 'NoneType' object has no attribute 'fullname': 
# pwnxy @ function > q